app/assets/stylesheets/application.scss

```diff
@@ -10,6 +10,29 @@
  * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
  * file per style scope.
  *
- *= require_tree .
- *= require_self
  */
+
+ @import "bootstrap-sprockets";
+ @import "bootstrap";
+
+.button_to {
+  display: inline-block;
+}
+
+#upvote {
+  display: block;
+  margin-right: 1200px;
+  margin-bottom: 20px;
+}
+
+.page-header {
+  background: url(http://www.picgifs.com/glitter-gifs/h/hello-kitty/picgifs-hello-kitty-6615712.gif);
```

Comment:
When using images it is safer to download the image and reference to it's folder location rather than rely on a 3rd party website. The image may get taken down or moved by the owner of the site or even worse could be swapped out for something unsavory.